app/views/account/_prefs_appearance.html.erb

```diff
@@ -0,0 +1,49 @@
+<!-- Appearance Settings of Account Preferences Form -->
+<div class="form-group push-down bold">
+  <%= :prefs_appearance.t %>
+</div>
+
+<div class="form-group push-down form-inline">
+  <%= label_tag(:user_hide_authors, :prefs_hide_authors.t + ":") %>
+  <%= form.select(:hide_authors,
+                  [[:prefs_hide_authors_none.l, :none],
+                   [:prefs_hide_authors_above_species.l, :above_species]],
+                  class: "form-control") %><br/>
+
+  <%= label_tag(:user_location_format, :prefs_location_format.t + ":") %>
+  <%= form.select(:location_format,
+                  [[:prefs_location_format_postal.l, :postal],
+                   [:prefs_location_format_scientific.l, :scientific]],
+                  class: "form-control") %><br/>
+
+  <%= label_tag(:user_theme, :prefs_theme.t + ":") %>
+  <%= form.select(:theme,
+                  [
```

Comment:
Removed extra white space (blank lines) within the div.  I think that's what you meant, but the way in which GitHub displays comments makes it unclear.  Perhaps you meant only remove the one instance of a double blank line.